src/components/TabSelector.js

```diff
@@ -0,0 +1,62 @@
+import {View} from 'react-native';
+import React from 'react';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import * as Expensicons from './Icon/Expensicons';
+import ONYXKEYS from '../ONYXKEYS';
+import TabSelectorItem from './TabSelectorItem';
+import Tab from '../libs/actions/Tab';
+import CONST from '../CONST';
+import useLocalize from '../hooks/useLocalize';
+import styles from '../styles/styles';
+import * as IOU from '../libs/actions/IOU';
+
+const propTypes = {
+    /** Which tab has been selected */
+    tabSelected: PropTypes.string,
```

Comment:
NAB I prefer `selectedTab`

```suggestion
    selectedTab: PropTypes.string,
```